README.md

```diff
@@ -77,6 +77,22 @@ this driver supports the following:
 > **Note**
 > `msedgedriver` support is limited. `appium:autodownloadEnabled` does not work for the driver, thus `appium:executable` is necessary to automate MSEdge browser properly.
 
+## Scripts
+
+### install-chromedriver
+
+This script is used to install the given or latest stable version of chromedriver from
+[Chorme for testing](https://github.com/GoogleChromeLabs/chrome-for-testing)
+in `node_modules/appium-chromedriver/chromedriver` directory with `appium driver run chromium install-chromedriver`.
+
+As same as [appium-chromedriver](https://github.com/appium/appium-chromedriver),
+`CHROMEDRIVER_VERSION` and `CHROMELABS_URL` environment variables are available.
```

Comment:
do we also want to support older versions (below 115)?

Reply:
No for now (as me for this command)

Reply:
The testing repo has `113`, so strictly, `113`+ can work with this for now.

Reply:
then it makes sense to add a note about this limitation to the docs

Reply:
Yea, i have added it in https://github.com/appium/appium-chromium-driver/pull/366/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R84 right now.